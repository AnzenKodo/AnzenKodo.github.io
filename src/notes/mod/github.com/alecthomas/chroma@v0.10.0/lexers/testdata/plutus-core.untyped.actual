(program 1.0.0
    [
        (lam x (force x))
        (delay [[(builtin addInteger) (con integer 4)] (con integer 8)])
    ]
)
