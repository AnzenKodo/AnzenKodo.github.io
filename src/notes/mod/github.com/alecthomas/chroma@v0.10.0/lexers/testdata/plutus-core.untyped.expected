[
  {"type":"Punctuation","value":"("},
  {"type":"Keyword","value":"program "},
  {"type":"LiteralString","value":"1.0.0"},
  {"type":"Text","value":"\n    "},
  {"type":"Punctuation","value":"["},
  {"type":"Text","value":"\n        "},
  {"type":"Punctuation","value":"("},
  {"type":"Keyword","value":"lam "},
  {"type":"Name","value":"x"},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"("},
  {"type":"Keyword","value":"force "},
  {"type":"Name","value":"x"},
  {"type":"Punctuation","value":"))"},
  {"type":"Text","value":"\n        "},
  {"type":"Punctuation","value":"("},
  {"type":"Keyword","value":"delay "},
  {"type":"Punctuation","value":"[[("},
  {"type":"Keyword","value":"builtin "},
  {"type":"Name","value":"addInteger"},
  {"type":"Punctuation","value":")"},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"("},
  {"type":"Keyword","value":"con "},
  {"type":"KeywordType","value":"integer"},
  {"type":"Text","value":" "},
  {"type":"LiteralNumberInteger","value":"4"},
  {"type":"Punctuation","value":")]"},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"("},
  {"type":"Keyword","value":"con "},
  {"type":"KeywordType","value":"integer"},
  {"type":"Text","value":" "},
  {"type":"LiteralNumberInteger","value":"8"},
  {"type":"Punctuation","value":")])"},
  {"type":"Text","value":"\n    "},
  {"type":"Punctuation","value":"]"},
  {"type":"Text","value":"\n"},
  {"type":"Punctuation","value":")"},
  {"type":"Text","value":"\n"}
]
